kuryr_kubernetes/controller/service.py

```diff
@@ -112,7 +114,8 @@ def start(self):
             f = functools.partial(self.run_periodic_tasks, None)
             self.tg.add_timer(1, f)
 
-        self.health_manager.run()
+        eventlet.spawn(self.exporter.run)
```

Comment:
I wonder if you shouldn't use `self.tg.add_thread()`… Also would be nice to have strategy how to kill those when service is stopped.